compaction/compaction_manager.cc

```diff
@@ -1632,6 +1635,10 @@ future<> compaction_manager::perform_cleanup(owned_ranges_ptr sorted_owned_range
         co_return;
     }
 
+    if (found_maintenance_sstables) {
+        co_await perform_offstrategy(t);
+    }
```

Comment:
If repair is running still, it will produce sstables that will not be cleaned up.

Reply:
That's currently the admin's problem.
Since cleanup is triggered manually, it's supposed to run after repair / RBNO is finished.
Wasting efforts on synchronizing between them at this stage is an overkill IMO.
When we automate the whole process, the we'll make sure to run cleanup (on all eligible nodes) as part of a distributed state machine that will serialize it properly with regards to possible concurrent operations.